hack/boilerplate.go.txt

```diff
@@ -0,0 +1 @@
+// test
```

Comment:
What is this file good for?

Reply:
> What is this file good for?

Sorry, the above 3 files are added by accident. They are removed in the rebased one.